### PR TITLE
Improve visibility of association of Ingress Operator with routing

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1730886

The goal is improve how discoverable the content is via search engines. Not sure if this passes that threshold or not.

### CODE REVIEW

**Comment by jboxman** [comment]
@openshift/team-documentation PTAL, thanks.

File: modules/nw-scaling-ingress-controller.adoc [hunk 1]
```diff
@@ -5,33 +5,35 @@
...
+to 3 replicas:
 +
 ----
 $ oc patch -n openshift-ingress-operator ingresscontroller/default --patch '{"spec":{"replicas": 3}}' --type=merge
+ingresscontroller.operator.openshift.io/default patched
 ----
 
-. Verify the default `IngressController` has been updated with the desired
-number of replicas.
+. Verify the default `IngressController` has been updated with the expected
```
Comment (kalexand-rh) [inline modules/nw-scaling-ingress-controller.adoc]: s/Verify the/Verify that the
I think "expected" works. If you're interested in other opinions, the IBM Style guide suggests using "required" when you pull "desired," and I'd probably say "Verify that the default `IngressController" scaled to the number of replicas that you specified:"
> :no_entry_sign: desired adj
Do not use. Use “required” or “the <item> that you want.”

File: modules/nw-scaling-ingress-controller.adoc [hunk 2]
```diff
@@ -5,33 +5,35 @@
 [id="nw-ingress-controller-configuration_{context}"]
 = Scaling an Ingress Controller
 
-An Ingress Controller can be manually scaled up or down based on
-requirements such as the requirement to increase throughput. `oc`
-commands are used to scale the `IngressController` resource.
-The following procedure provides an example for scaling up the
-default `IngressController`.
+An Ingress Controller can be manually scaled up or down based on routing
```
Comment (kalexand-rh) [inline modules/nw-scaling-ingress-controller.adoc]: I'm not sure which keywords you're trying to load, but I might start with "Manually scale an Ingress Controller to meet routing performance or availability requirements..."

**Review by kalexand-rh — COMMENTED** [review]
Two suggestions, but it LGTM.

**Comment by jboxman** [comment]
@kalexand-rh thanks!

File: modules/nw-scaling-ingress-controller.adoc (hunk 1 above)
Comment (jboxman) [reply modules/nw-scaling-ingress-controller.adoc]: I like that suggestion, thanks!

**Pull request merged**